lib/darksky_weather/api/weather_analysis.rb

```diff
@@ -0,0 +1,85 @@
+# A set of methods for analyzing and collating weather data
```

Comment:
Will you need a method in this file to determine the precipitation type? ... ie. there were 2 inches of precipitation (type=rain) vs. 2 inches of precipitation (type=snow)

Reply:
`precipitation` always refers to "liquid rain" vs. `accumulation` that refers to snow.

Reply:
I saw this in the [docs](https://darksky.net/dev/docs) the other day so I want to be sure:

> `precipType` optional
> The type of precipitation occurring at the given time. If defined, this property will have one of the following values: "rain", "snow", or "sleet" (which refers to each of freezing rain, ice pellets, and “wintery mix”). (If precipIntensity is zero, then this property will not be defined. Additionally, due to the lack of data in our sources, historical precipType information is usually estimated, rather than observed.)

Reply:
Hmm...they don't sound too convinced about the reliability of their own historical data, but to the extent that we ever do need it, it will be there for us. Using the gem, to get to this you'd only need to call something like `weather_data.daily.first.precip_type`. The inclusion of the method/data is automatic.

Reply:
Ok cool just wanted to make sure we could get the info easily enough!